docker-compose.elastic7.yml

```diff
@@ -1,12 +1,12 @@
 version: '3.0'
 services:
-  es7:
-    container_name: es7
+  es1:
```

Comment:
Doesn't this is in conflict for people that my want to run both at the same time as an migration step? So es5 and es7.

Reply:
That's true, however you can still start/stop the containers when testing
both.

With the current approach you need to change the container names in the
docker files and rebuild them so there's an extra step there.